src/places/models/place.model.ts

```diff
@@ -51,6 +51,10 @@ export class Place extends Model {
   @Column({ allowNull: true, type: DataType.DECIMAL })
   longitude!: number | null;
 
+  @ApiProperty({ nullable: true, type: 'string' })
+  @Column({ allowNull: true, type: DataType.STRING })
+  workingHours!: number | null;
```

Comment:
DB and DTO defines it as string. This place says number. I guess you wanted to type it as string
```suggestion
  workingHours!: string | null;
```

Reply:
Nice catch! Must have been some copy-paste glitch 😃